core/backend/src/AliCloudStorageService.ts

```diff
@@ -8,7 +8,12 @@ import { BentleyStatus } from "@itwin/core-bentley";
 import { CloudStorageContainerDescriptor, CloudStorageContainerUrl, CloudStorageProvider, IModelError } from "@itwin/core-common";
 import { CloudStorageService, CloudStorageUploadOptions } from "./CloudStorageBackend";
 
-/** @beta */
+/* eslint-disable deprecation/deprecation */
+
+/**
+ * @beta
+ * @deprecated
```

Comment:
What should people use instead now that this is deprecated?

Reply:
they should import the `@itwin/object-storage-oss` and `@itwin/object-storage-s3` package and pass an instance of `S3ServerStorage` to `IModelHost`... It would be easiest to use inversify to handle the dependency injection, but you could also pass your own instances of `S3PresignedUrlProvider` and `OssTransferConfigProvider` to the `S3ServerStorage` constructor.

However, as far as I know the existing implementation never worked properly anyway.

Reply:
Is there a way to indicate that succinctly as part of the `@deprecated` tag? Getting a "deprecated" warning when compiling your previously-working code with no indication of what you're supposed to do about it is very frustrating.

Reply:
The best I can think of without being too verbose is:
``@deprecated Use `@itwin/object-storage-oss` instead``
I guess that's better than nothing.

Reply:
I'd say `Use @itwin/object-storage-oss with @itwin/object-storage-s3 instead for AliCloud support` or something similar. Doesn't have to be too short.

Reply:
if you use inversify, you can just use `@itwin/object-storage-oss` and it will import what it needs from the other package. It exposes its own DI bindings but doesn't re-export the S3 implementations that it binds. I also think the object-storage repo includes better instructions for using inversify, so that should be the recommended path.

Reply:
@calebmshafer resolve if you're satisfied.

Reply:
that sounds good to me.